source/navitiacommon/navitiacommon/models.py

```diff
@@ -263,6 +265,45 @@ def __repr__(self):
         return '<Instance %r>' % self.name
 
 
+class TravelerProfile(db.Model):
+
+    coverage = db.Column(db.Text, db.ForeignKey('instance.name'), primary_key=True, nullable=True)
```

Comment:
according to the spec, default profiles are hard coded in jormun's code: https://github.com/xlqian/navitia/blob/23afb14e308df5b336cd47b3f861da484ece47c5/source/jormungandr/jormungandr/travelers_profile.py#L136